ethsigner/src/main/java/tech/pegasys/ethsigner/jsonrpc/response/JsonRpcError.java

```diff
@@ -88,7 +88,10 @@
   // Private transaction errors
   ENCLAVE_IS_DOWN(-50100, "Enclave is down"),
   UNIMPLEMENTED_PRIVATE_TRANSACTION_TYPE(-50100, "Unimplemented private transaction type"),
-  PRIVATE_TRANSACTION_RECEIPT_ERROR(-50100, "Error generating the private transaction receipt");
+  PRIVATE_TRANSACTION_RECEIPT_ERROR(-50100, "Error generating the private transaction receipt"),
+
+  // Gateway errors
+  DOWNSTREAM_NODE_TIMED_OUT(-32000, "Downstream node timed out");
```

Comment:
`Downstream node timed out` -> `Connection to downstream node timed out`

Reply:
done